packages/frontend/src/features/semanticViewer/runners/SemanticViewerResultsRunner.ts

```diff
@@ -174,27 +174,22 @@ export class SemanticViewerResultsRunner extends ResultsRunner {
         fields: SemanticLayerField[],
         columns: string[],
     ): VizColumn[] {
-        return columns
-            .map<VizColumn | undefined>((column) => {
-                const field =
-                    SemanticViewerResultsRunner.findSemanticLayerFieldFromColumn(
-                        fields,
-                        column,
-                    );
-                if (!field) {
-                    return;
-                }
```

Comment:
bug

Reply:
this is because of the columns that come back in the pivoted table correct?

Reply:
yep!